platforms/core-runtime/logging/src/main/java/org/gradle/internal/deprecation/DeprecationMessage.java

```diff
@@ -28,18 +28,27 @@ class DeprecationMessage {
     private final String context;
     private final DocLink documentation;
     private final DeprecatedFeatureUsage.Type usageType;
-
-    DeprecationMessage(String summary, String removalDetails, @Nullable String advice, @Nullable String context, @Nullable DocLink documentation, DeprecatedFeatureUsage.Type usageType) {
+    private final String problemId;
+
+    DeprecationMessage(String summary,
```

Comment:
```suggestion
    DeprecationMessage(
        String summary,
```

💅  Please put the first argument on a new line for better diffability in the future.